src/commonMain/kotlin/org/kosat/heuristics/PriorityQueue.kt

```diff
@@ -0,0 +1,121 @@
+package org.kosat.heuristics
+
+class PriorityQueue {
```

Comment:
Why not just use [`java.util.PriorityQueue`](https://docs.oracle.com/javase/7/docs/api/java/util/PriorityQueue.html)?